code/game/machinery/mass_driver.dm

```diff
@@ -156,6 +158,8 @@
 /obj/machinery/mass_driver_frame/screwdriver_act(mob/living/user, obj/item/I)
 	if(build != 4)
 		return
+	if(check_screw_size())
```

Comment:
you're missing args here

Reply:
Not even the only place where I missed them, so a really good catch.